tests/integration/integration_test.go

```diff
@@ -186,7 +186,7 @@ func checkNewContainers(t *testing.T, gotOutput *eventOutput) {
 	containerIds := []string{}
 	output := gotOutput.getEventsCopy()
 	for _, evt := range output {
-		containerIds = append(containerIds, evt.Container.ID)
+		containerIds = append(containerIds, evt.Container.ID, evt.ContainerID)
```

Comment:
Why do we add (the same) container id twice here?

Reply:
Just confirming the field duplication works as intended.

Reply:
Once we move to the new type I assume the test will break and change again.